packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -107,47 +95,47 @@ When a specific client/IP/device identifier combination exceeds either the 60 re
+    * Select **Log per client** to enable client-based rate limit in preview mode. In **Log per client** mode, rate limit is based on the org-wide rate limit values, but client-specific rate limit error information is logged as System Log events. By analyzing these System Log events, you can determine if client-based rate limit is effective for you.
 
-    * Select **No action** to disable client-based rate limiting.
+    * Select **No action** to disable client-based rate limit.
 
 ### Frequently asked questions
 
-**Q: Which endpoints are covered under client-based rate limiting?**
+**Q: Which endpoints are covered under client-based rate limit?**
 
-Currently, client-based rate limiting only applies to an authorization server's `/authorize` endpoint.
+Currently, a client-based rate limit only applies to an authorization server's `/authorize` endpoint.
 
 **Q: How is the client-specific rate limit determined?**
 
-The client rate-limiting framework calculates the per client rate limit based on the OAuth client ID, the user's IP address, and the Okta device identifier (the Okta device identifier that Okta sets in the browser).
+The client rate limit framework calculates the per client rate limit based on the OAuth client ID, the user's IP address, and the Okta device identifier (the Okta device identifier that Okta sets in the browser).
 
 **Q: What happens if my network contains a proxy server through which the requests are proxied?**
 
-Requests would appear to come from the same IP Address. When `/authorize` requests are made from behind a proxy IP address, make sure to [configure the respective IPs as proxies](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Security_Network). This allows the client-based rate-limiting framework to look for the IP address before the proxy to find the true client IP address.
+Requests would appear to come from the same IP Address. When `/authorize` requests are made from behind a proxy IP address, make sure to [configure the respective IPs as proxies](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Security_Network). This allows the client-based rate-limit framework to look for the IP address before the proxy to find the true client IP address.
 
-**Q: Can I update the per client rate limiting today?**
+**Q: Can I update the per client rate limit today?**
 
 No. Today every client ID/IP/device identifier combination gets 60 total requests per minute and a maximum of two concurrent requests.
 
-**Q: Does the org-wide rate limit still apply when I enable client-based rate limiting?**
+**Q: Does the org-wide rate limit still apply when I enable client-based rate limit?**
```

Comment:
"enable client-based rate limit" 
-> "enable the client-based rate limit"
or -> "enable client-based rate limiting"

Reply:
Fixed.